src/js/notifications.js

```diff
@@ -0,0 +1,67 @@
+var Wand = (function(wand) {
+
+/* Inspiration taken from
+   https://github.com/cugos/dropchop/blob/master/src/js/notifications/Notifications.js
+ */
+
+  'use strict';
+
+  wand = wand || {};
+  wand.notifications = {};
+  var params;
+
+  wand.notifications.init = function() {
```

Comment:
Do we want to have some sort of domElement initialization that builds all of the pieces at once instead of pushing them in different pieces of the code base? Right now we are making DOM elements in two places, `wand.js init()` function and this file.